src/NewEmail/Form/index.jsx

```diff
@@ -0,0 +1,103 @@
+// Note for the next TEAM:1. For function's you have console.log() to see in console what is happening
+// Note for the next TEAM:2. In this Form component you will find The Form and the Buttons
+// Note for the next TEAM:3. You will find comments before Function for what are they and so on
+import {useState} from "react"
+function Form() {
+    const [email, setEmail] = useState("")
+    const [subject, setSubject] = useState("")
+    const [textArea, setTextArea] = useState("")
+    const [send, setSend] = useState(false)
+    const [cancel, setCancel] = useState(false)
+
+    // Function for cancel button ->
+    const handleCancel = () => {
+        setEmail("")
+        setSubject("")
+        setTextArea("")
+        setCancel(true)
+        setSend(false)
+        console.log("Cancel action ON")
+    }
+
+    // Function for Send button ->
+    const handleSend = () => {
+        if (email.trim() && subject.trim() && textArea.trim()) {
+            setSend(true)
+            setCancel(false)
+
+            // Data sending to API
+            fetch("https://email-client-api.dev.io-academy.uk/email", {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                body: JSON.stringify({
+                    to: email,
+                    subject: subject,
+                    content: textArea
+                })
+            })
+                .then((response) => response.json())
+                .then((data) => {
+                    console.log("Answer API:", data)
+                    setSend(true) // confirmation for sent data
+                })
+                .catch((error) => {
+                    console.error("Error for data sent", error)
+                    setSend(false)
+                })
+        } else {
+            console.log("Complete forms before sending")
+        }
+    }
+
+    return (
+        <div className=" text-black max-w-sm p-10 ">
+            <form className="flex flex-col space-y-4 gap-3 ml-3">
+                <label>
+                    <input
+                        placeholder="To"
+                        type="text"
+                        value={email}
+                        onChange={(e) => setEmail(e.target.value)}
+                        className="w-96 p-1.5 border border-gray-300 rounded focus:outline-none focus:ring-2 focus:ring-gray-400 placeholder-gray-500"/>
+                </label>
+                <label>
+                    <input
+                        placeholder="Subject"
+                        type="text"
+                        value={subject}
+                        onChange={(e) => setSubject(e.target.value)}
+                        className="w-96 p-1.5 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-gray-400 placeholder-gray-500"/>
```

Comment:
These two inputs are the same, you should make a text input component and re-use it for both of these. You can pass in all the data (placeholder, value, onChange function etc) as props.

That way, if you ever need more inputs elsewhere, you can just re-use the component. Also, if you ever need to change the way they look, you only have to change it in one place.

Reply:
Also - can you make these inputs full width please, I think it would be better if they took up the whole width of the page.